packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -271,7 +271,7 @@ class ThemeData with Diagnosticable {
     FloatingActionButtonThemeData floatingActionButtonTheme,
     NavigationRailThemeData navigationRailTheme,
     Typography typography,
-    CupertinoThemeData cupertinoOverrideTheme,
+    NoDefaultCupertinoThemeData cupertinoOverrideTheme,
```

Comment:
You can still pass in a CupertinoThemeData because its a subclass of NoDefaultCupertinoThemeData. This shouldn't break anybody, I think.